### PR TITLE
Feature/choro

### PR DESCRIPTION
**Pull request by BennuFire**
Choropleth and More

### CODE REVIEW

File: src/config/ReportConfig.tsx [hunk 1]
```diff
@@ -915,6 +916,102 @@ export const REPORT_TYPES = {
...
+                default: true
+            },
+            "colors": {
+                label: "Color Scheme",
+                type: SELECTION_TYPES.LIST,
+                values: ["nivo", "category10", "accent", "dark2", "paired", "pastel1", "pastel2", "set1", "set2", "set3", "YlOrRd"],
+                default: "nivo"
+            },
+            "borderWidth": {
+                label: "Rectangle border width (px)",
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply src/config/ReportConfig.tsx]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: src/config/ReportConfig.tsx [hunk 2]
```diff
@@ -915,6 +916,102 @@ export const REPORT_TYPES = {
...
+
+            },
+            "key": {
+                label: "code",
+                type: SELECTION_TYPES.TEXT,
+                optional: true
+            }
+        },
+        useRecordMapper: true,
+        maxRecords: 3000,
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply src/config/ReportConfig.tsx]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: src/chart/visualizations/ChoroplethMapVisualization.tsx [hunk 3]
```diff
@@ -0,0 +1,797 @@
...
+} from './Utils'
+import { useState } from 'react'
+import { Tooltip } from '@material-ui/core'
+import RefreshIcon from '@material-ui/icons/Refresh';
+import { globeFeature, colombiaFeature } from '../util/ChoroplethFeatures'
+
+export default function ChroplethMapVisualization(props: ExtendedChartReportProps) {
+    const { records, first } = props
+
+    const dataTestChoropleth = [
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply src/chart/visualizations/ChoroplethMapVisualization.tsx]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+[width="100%",cols="15%,2%,26%,57%",options="header",]
+|===
+|Name |Type |Default Value |Description
+|Enable interactivity |on/off |on |If enabled, turn on animations when a
+user hovers over a rectangle.
+
+|Color Scheme |List | |The color scheme to use for the choropleth. Country colors
+will vary according their min to max ratio.
+
+|Rectangle border width (px) |number |0 |The width of the border of each
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+image::choropleth.png[Treemap Chart]
+
+== Advanced Settings
+
+[width="100%",cols="15%,2%,26%,57%",options="header",]
+|===
+|Name |Type |Default Value |Description
+|Enable interactivity |on/off |on |If enabled, turn on animations when a
+user hovers over a rectangle.
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+[source,cypher]
+----
+MATCH (:Company{name:'NeoDash'})-[:HAS_DEPARTMENT]->(:Department)<-[:IN_DEPARTMENT]-(e:Employee),
+(e)-[:LIVES_IN]->(c:Country)
+WITH c.code as code, count(e) as value
+RETURN code, value
+----
+
+image::choropleth.png[Treemap Chart]
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply docs/modules/ROOT/pages/user-guide/reports/choropleth.adoc]: Solved in https://github.com/neo4j-labs/neodash/pull/174/commits/df82e724a9dd6d9047b89210dc556c0d517a4589

File: src/chart/visualizations/ChoroplethMapVisualization.tsx [hunk 7]
```diff
@@ -0,0 +1,797 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (BennuFire) [reply src/chart/visualizations/ChoroplethMapVisualization.tsx:1]: https://github.com/neo4j-labs/neodash/pull/174/commits/1d46fc3d76d55d944e55e15826afd0a8de48a5f8

**Comment by BennuFire** [comment]
Updated

**Review by nielsdejong — APPROVED** [review]

**Pull request merged**